src/member/templates/client/view/notes.html

```diff
@@ -46,6 +46,7 @@ <h2 class="ui header">
         <div class="field">
             <a href="{{ request.path }}?display={{ display }}" class="ui button">{% trans "Reset" %}</a>
             <button class="ui yellow button" type="submit">{% trans "Search" %}</button>
+            <a href="{% url 'member:client_notes_add' client.pk %}" class="ui primary">{% trans "Add a Note" %}</a>
```

Comment:
Will be corrected in next commit:
![image](https://cloud.githubusercontent.com/assets/8630726/20840751/4b47280c-b87f-11e6-95a2-bfa64db41622.png)